### PR TITLE
chore: setup local environment using firebase emulators 

### PR DESCRIPTION
**Pull request by yarn-rp**
## Description
Unify both repositories on the same one, and setups local environment using firebase emulators. 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,113 +1,88 @@
...
 
-Blueprint is not yet ready for production, but you can install it in your machine and try it out. To install it, you need to have Flutter installed in your machine. You can follow the instructions [here](https://flutter.dev/docs/get-started/install). Once you have Flutter installed, you can clone this repository.
+2. Navigate to the project directory.
+```
+cd blueprint
+```
+
+3. Install the required node packages.
+```
+npm install
```
Comment (jvaldesgonzalez) [inline README.md:35]: suggestion: yarn is recommended

Comment (jvaldesgonzalez) [reply README.md:35]: also `yarn install` must be executed inside functions folder

File: README.md [hunk 2]
```diff
@@ -1,113 +1,88 @@
...
+_\*Blueprint has official support only for web._
 
-There you can add the integrations you want. Currently supports only Jira and GitHub, but more integrations are planned to be added in the future. Once you have added the integrations you want, you can go back to the might do screen and you will see your tasks there:
+#### Local environment
 
-TODO: Insert image of the might do screen with tasks
+There is another env called local which will use a combination of the development service account, and local emulators so we can test in a more isolated environment. This is the default environment to run for developers. 
 
-### Might do
+First, we are going to serve the firebase project running via emulators. The emulators config can be found on `firebase.json` file at the root of the project. If your local machine uses the declared ports for other functions, you can change these ports for some available ports on your machine. Just make sure that you **do not commit changes on `firebase.json` file** after it.
```
Comment (jvaldesgonzalez) [inline README.md:70]: suggestion: isn't clear that we must copy local.env to .env. Add "run command `cp local.env .env` inside functions folder", or add command to the yarn script

File: README.md [hunk 3]
```diff
@@ -1,113 +1,88 @@
...
+This is a task manager for multiple projects. It allows you to manage your tasks in one place, even if they are in different platforms. It is built with Flutter, so it can run on web for now, but other platforms support are about to come. 
 
-## Why using it?
+## Getting started
 
-If you are a productivity freak like me, you probably have tasks in different platforms. For example, you may have tasks in Jira, Trello, GitHub, etc. It is hard to keep track of all your tasks, specially if you are working at the same time in different projects, cause not all of them are in the same platform, and sometimes the same platform is used for different projects. Blueprint allows you to manage all your tasks in one place, by integrating with the different platforms you use. It is not a replacement for those platforms, it is just a way to keep track of all your tasks in one place.
+Before setting up the project, you need to have the following tools installed:
 
-## Data privacy
+- **Node.js** runtime environment (version 8 or higher)
```
Comment (jvaldesgonzalez) [inline README.md:17]: this is outdated, the recommended node version is 16.x.x

File: functions/scripts/seed.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import * as admin from "firebase-admin";
+import * as fs from "fs";
+import * as path from "path";
+
+const LOCALHOST = "localhost";
+
+const firebaseConfigPath = path.join("./../", "firebase.json"); // adjust the path as needed
```
Comment (jvaldesgonzalez) [inline functions/scripts/seed.ts:7]: suggestion(non-blocking): this isn't the right way, add "resolveJsonModule": true into compilerOptions in tsconfig file. Then use the normal import syntax

**Review by jvaldesgonzalez — CHANGES_REQUESTED** [review]

File: functions/scripts/seed.ts (hunk 4 above)
Comment (yarn-rp) [reply functions/scripts/seed.ts:7]: I didn't like it either, but didn't want to mess that much with the config. Glad to know that there is a better non ugly way.

File: README.md (hunk 3 above)
Comment (yarn-rp) [reply README.md:17]: noted

File: README.md (hunk 1 above)
Comment (yarn-rp) [reply README.md:35]: thanks for spotting the wrong directory!

File: README.md (hunk 2 above)
Comment (yarn-rp) [reply README.md:70]: Yeah, I missed this step. Thanks for noticing it.

Comment (yarn-rp) [reply README.md:78]: Let me know if thats clear enough or if you wanna add something to it

File: functions/scripts/seed.ts [hunk 5]
```diff
@@ -1,14 +1,9 @@
 import * as admin from "firebase-admin";
-import * as fs from "fs";
-import * as path from "path";
-
+import * as firebaseConfig from "./../../firebase.json";
 const LOCALHOST = "localhost";
```
Comment (jvaldesgonzalez) [inline functions/scripts/seed.ts:3]: nit: change LOCALHOST to simply HOST

**Review by jvaldesgonzalez — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jvaldesgonzalez) [reply README.md:78]: Seems great!

**Pull request merged**